[2022-03-11 18:54:08,354] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_04 scheduled__2022-03-10T00:00:00+00:00 [queued]>
[2022-03-11 18:54:08,682] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_04 scheduled__2022-03-10T00:00:00+00:00 [queued]>
[2022-03-11 18:54:08,684] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-11 18:54:08,685] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-03-11 18:54:08,685] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-11 18:54:08,785] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_04> on 2022-03-10 00:00:00+00:00
[2022-03-11 18:54:08,789] {standard_task_runner.py:52} INFO - Started process 7258 to run task
[2022-03-11 18:54:08,802] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_04', 'scheduled__2022-03-10T00:00:00+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dags.py', '--cfg-path', '/tmp/tmp2tlplc_9', '--error-file', '/tmp/tmp04icdkyg']
[2022-03-11 18:54:08,804] {standard_task_runner.py:77} INFO - Job 79: Subtask task_04
[2022-03-11 18:54:09,482] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_04 scheduled__2022-03-10T00:00:00+00:00 [running]> on host airflowvm
[2022-03-11 18:54:10,587] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_04
AIRFLOW_CTX_EXECUTION_DATE=2022-03-10T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-10T00:00:00+00:00
[2022-03-11 18:54:10,588] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-11 18:54:10,593] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-03-11 18:54:10,696] {subprocess.py:85} INFO - Output:
[2022-03-11 18:54:13,704] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-11 18:54:14,234] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_04, execution_date=20220310T000000, start_date=20220311T185408, end_date=20220311T185414
[2022-03-11 18:54:14,344] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-11 18:54:14,539] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
